microlith/nginx/nginx.conf

```diff
@@ -67,6 +67,11 @@ http {
         location /licenses/ {
             autoindex on;
         }
+
+        location /support {
+            alias /tmp/support/;
+            autoindex off; # avoid accidental leaks
```

Comment:
Does this mean we cannot download them through the browser? I want to make the UX for that simple, we should not need to know the full name/path to download it really.

Reply:
True, I can flip that back - you can download them through the browser but only if you know the name - I wanted to have it off to avoid it being accidentally exposed, but now that I think about it this will be firewalled off anyway so it makes sense.